chain/indexer/src/lib.rs

```diff
@@ -50,6 +49,32 @@ impl Indexer {
         Self { actix_runtime: system, view_client, client, near_config }
     }
 
+    /// Initializes genesis and client configs and stores in the given folder
+    /// Exposes `neard::init_configs()`
+    pub fn init_configs(
```

Comment:
This looks suboptimal. Why do you need to do this?

Reply:
Honestly, I have no idea, I'm already fixing it :)